processing_chain/src/test/groovy/org/orbisgis/processingchain/ProcessingChainTest.groovy

```diff
@@ -0,0 +1,30 @@
+package org.orbisgis.processingchain
+
+import org.junit.jupiter.api.Test
+
+import org.orbisgis.datamanager.h2gis.H2GIS
+import org.orbisgis.processmanager.ProcessMapper
+
+class ProcessingChainTest {
+
```

Comment:
In order to skip this test when the resources are not present (as the geoclimate_bdtopo_data_test should not be published) add the following method which register a system property which will be used to disable test if needed.
``` groovy
@BeforeAll
static void init(){
    System.setProperty("test.processingchain",
            Boolean.toString(ProcessingChainTest.getResource("geoclimate_bdtopo_data_test") != null))
}
```